tests/auth/test_controller.py

```diff
@@ -267,4 +269,123 @@ def tearDown(self):
         with self.app.app_context():
             db.session.remove()
             db.drop_all()
-            db.engine.dispose()
\ No newline at end of file
+            db.engine.dispose()
+
+
+class AuthorizeUploadTestCase(unittest.TestCase):
```

Comment:
@rufuspollock removed `GetS3SignedUrlTestCase` in commit 7cb0748b9f099fa0a87b3231cfffb66643b31f62